freeipa/src/main/java/com/sequenceiq/freeipa/util/KrbKeytabGenerator.java

```diff
@@ -0,0 +1,54 @@
+package com.sequenceiq.freeipa.util;
```

Comment:
I'm not too fond of the location of this file. I'd prefer it to reside closer to the service that will actually use it.

Reply:
Ah sure let me find a better place (I've placed it here because KrbKeySetEncoder is here which plays an analogous role). I think we want to expose this as part of UserV1Endpoint and maybe add a UserKeytabGenService.

Reply:
that class is in a wrong package also IMHO

Reply:
I gree that KrbKeySetEncoder is also in the wrong spot. I suggest com.sequenceiq.freeipa.service.freeipa.user.krb or com.sequenceiq.freeipa.service.freeipa.user.util

Reply:
Thanks, I'll move both of them.